src/sentry/utils/prompts.py

```diff
@@ -2,6 +2,7 @@
     "releases": {"required_fields": ["organization_id", "project_id"]},
     "suspect_commits": {"required_fields": ["organization_id", "project_id"]},
     "alert_stream": {"required_fields": ["organization_id"]},
+    "app_store_connect_updates": {"required_fields": ["organization_id", "project_id"]},
```

Comment:
I have no idea what this is.  But assume it's good...

Reply:
this is to suspend an alert message for a certain time